tests/PKSim.Tests/Infrastructure/DomainFactoryForSpecs.cs

```diff
@@ -54,6 +55,24 @@ public static Protocol CreateStandardIVProtocol()
          return protocolFactory.Create(ProtocolMode.Simple, ApplicationTypes.Intravenous).WithName("Protocol");
       }
 
+      public static Protocol CreateStandardOralProtocol()
+      {
+         var protocolFactory = IoC.Resolve<IProtocolFactory>();
+         return protocolFactory.Create(ProtocolMode.Simple, ApplicationTypes.Oral).WithName("Protocol");
+      }
+
+      public static Formulation CreateParticlesFormulation(int numberOfBins)
+      {
+         var formulationRepository = IoC.Resolve<IFormulationRepository>();
+         var formulation = formulationRepository.FormulationBy(CoreConstants.Formulation.Particles);
+         formulation.Parameter(CoreConstants.Parameter.NUMBER_OF_BINS).Value = numberOfBins;
+         
+         //set mono/polydisperse property (0=mono, 1=poly).
+         formulation.Parameter(CoreConstants.Parameter.PARTICLE_DISPERSE_SYSTEM).Value = (numberOfBins > 1) ? 1 : 0;
+
+         return formulation;
+      }
+
       public static IndividualSimulation CreateDefaultSimulation()
```

Comment:
New funcxtionality required for testing of simulation with particles dissolution